### ISSUE TITLE
Build error on void * return 

### ISSUE THREAD

**Issue by avindra** [Sweets/custard#2]
packaging custard fails with openSUSE's checks, which is throwing:

```
 I: Program returns random data in a function
 E: custard no-return-in-nonvoid-function src/custard.cpp:6
```

It's coming from this function, which does not return anything. The type is `void *` so it _should_ return something:

https://github.com/Sweets/custard/blob/5cb47fab81410196b0e3407ef4f70e767f4bbc89/src/custard.cpp#L3-L6

**Comment by Sweets** [comment]
Should be good now, let me know.

**Comment by avindra** [comment]
Yep, it builds now:

https://build.opensuse.org/package/show/home:avindra/custard

Thanks!

**Comment by Sweets** [comment]
Small heads up that 1.2 was just finished and can now be downloaded as a release, so the package will no longer need to use a patched version of 1.1.
